packages/core/test/Subsidy.ts

```diff
@@ -0,0 +1,230 @@
+// @ts-ignore
+import { ethers } from 'hardhat'
+import { expect } from 'chai'
+import { ZkIdentity } from '@unirep/crypto'
+import { genEpochKey } from '@unirep/core'
+import { genUserState } from './utils'
+import { deployUnirep } from '@unirep/contracts/deploy'
+import { deployUnirepSocial } from '../src/utils'
+import * as config from '@unirep/circuits'
+
+describe('Subsidy', function () {
```

Comment:
Should add tests to see after user state transition
if user can receive reputation with the subsidy epoch key.
and after epoch transition user can request more subsidy for the next epoch

Reply:
and neg reputation users should successfully get airdrop and come back to the system

Reply:
e09f9de74f79279a8bc77ca10dbb18bdbb57b820